src/components/structures/MatrixChat.tsx

```diff
@@ -1193,6 +1198,17 @@ export default class MatrixChat extends React.PureComponent<IProps, IState> {
         });
     }
 
+    private async copyRoom(roomId: string) {
+        const roomLink = makeRoomPermalink(roomId);
+        const success = await copyPlaintext(roomLink);
+        if (!success) {
+            Modal.createTrackedDialog("Unable to copy room", "", ErrorDialog, {
+                title: _t("Unable to copy room"),
+                description: _t("Unable to copy the room to the clipboard."),
```

Comment:
this should probably be `room link` as otherwise the error might be suggesting that you're copying the entire room to the clipboard, where an error would be way more expected.